entity-registry/src/main/java/com/linkedin/metadata/models/annotation/AspectAnnotation.java

```diff
@@ -16,11 +17,15 @@
   public static final String ANNOTATION_NAME = "Aspect";
   public static final String NAME_FIELD = "name";
   private static final String TYPE_FIELD = "type";
+  private static final String RENDER_SPEC_FIELD = "renderSpec";
```

Comment:
These new annotation types need to be documented on the https://datahubproject.io/docs/metadata-modeling/extending-the-metadata-model#metadata-annotations section.